packages/forms-web-app/src/services/pdf.service.js

```diff
@@ -0,0 +1,33 @@
+const fetch = require('node-fetch');
+const { createDocument } = require('../lib/documents-api-wrapper');
+const { generatePDF } = require('../lib/pdf-api-wrapper');
+const { VIEW } = require('../lib/views');
+
+const getHtmlAppeal = async (appeal) => {
+  const response = await fetch(
+    `http://forms-web-app:3000/${VIEW.APPELLANT_SUBMISSION.SUBMISSION_INFORMATION}/${appeal.id}`
```

Comment:
hardcoded to `http://forms-web-app:3000` - will that work on k8s? (I have no idea btw)

Reply:
I think so as there is internal services